src/frontend/utils/tests/imageSnapshot.tsx

```diff
@@ -0,0 +1,44 @@
+import { cleanup, render } from '@testing-library/react';
+import { generateImage } from 'jsdom-screenshot';
+import { Grommet } from 'grommet';
+import React from 'react';
+import path from 'path';
+
+import { wrapInIntlProvider } from './intl';
+import { wrapInRouter } from './router';
+import { theme } from '../theme/theme';
+import { GlobalStyles } from '../theme/baseStyles';
+
+export const imageSnapshot = async (width?: number, height?: number) => {
```

Comment:
If I understand, this function will generate an image of the current DOM in jsdom ?

Reply:
Exactly, and we can add it multiple times on the same test, taking image snapshots of current state.

Reply:
Ok so functions after this one are helpers to use this one. I didn't understand this in the Readme. You can maybe modify the readme to explain it ?

Reply:
Ok, sorry for being confusing.